src/common/options.cc

```diff
@@ -3169,6 +3169,14 @@ std::vector<Option> get_global_options() {
     .set_default(-1)
     .set_description(""),
 
+    Option("bdev_enable_discard", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(true)
+    .set_description(""),
+
+    Option("bdev_async_discard", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(true)
+    .set_description(""),
+
```

Comment:
Should we set both options to true by default? Don't we want to disable this by default and let users enable it if they want to?

Reply:
@wido 
I set the flag as true, for checking the correctness of this option in jenkins test.
I agree with you.
In addition to changing the default value, some variables' names will be also modified.
Thanks for reviewing it.

Reply:
Ok, understood. Just checking :)

We might want to prefix these with *bluestore* indeed as they apply to BlueStore only.

What do you say @liewegas ?

Reply:
I just thought this feature was for bdev class, specifically.
Reasonable suggestion! It's possible to change the option name as you said.
:)